common/setups/rasr/config/am_config.py

```diff
@@ -0,0 +1,143 @@
+__all__ = ["Tdp", "SpeechTdp", "SilenceTdp", "NonSpeechTdp", "AmRasrConfig"]
+
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import List, Optional, Tuple, Union
+
+
+from sisyphus import tk
+
+import i6_core.rasr as rasr
+
+
+class StateTying(Enum):
+    MONOPHONE = 1
+    CART = 2
+    DENSE = 3
+
+    def __str__(self):
+        if self.value == 1:
+            return "monophone"
+        elif self.value == 2:
+            return "cart"
+        elif self.value == 3:
+            return "dense"
```

Comment:
For dense tying there are three different cases (monophone, diphone and triphone), and one needs to specify also boundary class and word end class.

Reply:
> we can add other options

certainly, we can add different dense tyings, but then the word end class and boundary that is specific to the dense tying needs a separate enum class. I will take care of this when I integrate FH. working on it now.

Reply:
The options to add are:

`monophone-dense`, `diphone-dense`, and `no-tying-dense`, for now @christophmluscher

Reply:
I added all options from RASR